functions/Add-DbaAgDatabase.ps1

```diff
@@ -134,98 +134,106 @@ function Add-DbaAgDatabase {
             }
 
             if (-not $Secondary) {
-                $secondaryReplicas = $ag.AvailabilityReplicas | Where-Object Role -eq Secondary
-            } else {
-                $secondaryReplicas = Get-DbaAgReplica -SqlInstance $Secondary -SqlCredential $SecondarySqlCredential -AvailabilityGroup $ag.Name | Where-Object Role -eq Secondary
-            }
+                try {
+                    $secondaryInstances = ($ag.AvailabilityReplicas | Where-Object Role -eq Secondary).name | Connect-DbaInstance -SqlCredential $SecondarySqlCredential
+        } catch {
+            Stop-Function -Message "Failure connecting to secondary instance" -ErrorRecord $_ -Continue
+        }
+    } else {
+        $secondaryInstances = Connect-DbaInstance -SqlInstance $Secondary -SqlCredential $SecondarySqlCredential
+    }
 
-            if ($SeedingMode -ne "Automatic") {
-                if ((Get-DbaDatabase -SqlInstance $ag.Parent -Database $db.Name).LastFullBackup -eq 'Monday, January 1, 0001 12:00:00 AM') {
-                    Stop-Function -Message "Cannot add $($db.Name) to $($ag.Name) on $($ag.Parent.Name). An initial full backup must be created first." -Continue
-                }
-            }
+    if ($SeedingMode -ne "Automatic") {
+        if ((Get-DbaDatabase -SqlInstance $ag.Parent -Database $db.Name).LastFullBackup -eq 'Monday, January 1, 0001 12:00:00 AM') {
```

Comment:
Thanks for this PR! This particular code will cause issues on servers that are not based in English. Perhaps it can be cast to a mmddyy or just grab the year and compare 0001.

Reply:
This particular change was how I found the file when I picked it up - I think the formatting change has picked it up.